### PR TITLE
Rj/csw 184

### PR DESCRIPTION
**Pull request by rajnov08**
Changes to Units and related docs for recently added units in CSW.

### CODE REVIEW

**Comment by jasonweiss** [comment]
I think the description of the units in the docs should be more descriptive.  Simply using the number of the unit as description does give any extra info.  For example,  I don't know what an alpha is.  Ampere could be something like "measure of current".  We don't need to overdo it for things are the very commonly used, but some of the lesser known units could have some description of what they are.  

Another thing we might want to consider is using unicode for units with a special character, like "\u212b" for angstroms.  I think they are supported in the languages we use.  What do you think about this suggestion, @kgillies ?

**Comment by rajnov08** [comment]
I have made relevant changes to description of units. Do you want me to go ahead with unicode? if that is still to be discussed, can we merge this PR? @jasonweiss @kgillies

File: docs/src/main/params/units.md [hunk 1]
```diff
@@ -118,16 +118,16 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
+| seimens    | seimens            | derived unit of electric conductance |
+| tesla      | tesla              | unit of magnetic induction |
 | u          | u                  | atomic mass unit |
 
 ## Imperial Units
 
 | Name  | Abbreviation | Description                                          |
 | ----- | ------------ | ---------------------------------------------------- |
-| barn       | barn              | barn |
+| barn       | barn               |  metric unit of area |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: unit of area for cross section of scattering (10^28 m^2)

File: docs/src/main/params/units.md [hunk 2]
```diff
@@ -118,16 +118,16 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
 | rgeo       | Rgeo               | Earth radius |
 | rjup       | Rjup               | Jupiter radius |
 | rsun       | Rsun               | solar radius |
-| rydberg    | Rydberg            | Rydberg |
-| seimens    | seimens            | seimens |
-| tesla      | tesla              | tesla |
+| rydberg    | Rydberg            | unit of energy, Ry |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: ionization energy of hydrogen in its ground state (~13.6 eV)

File: docs/src/main/params/units.md [hunk 3]
```diff
@@ -83,8 +83,8 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
 | ---------- | ------------ | ------------------------ |
 | coulomb    | C                  | coulomb: electric charge |
 | centimeter | cm                 | centimeter |
-| D          | D                  | Debye(dipole) |
-| dyn        | dyn                | dyne |
+| D          | D                  | Debye(dipole) - electric dipole moments of molecules |
+| dyn        | dyn                | dyne is unit of force |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: CGS unit of force

File: docs/src/main/params/units.md [hunk 4]
```diff
@@ -83,8 +83,8 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
 | erg        | erg                | erg: CGS unit of energy |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md:88]: CGS unit of energy

Comment (jasonweiss) [inline docs/src/main/params/units.md]: CGS unit of energy

File: docs/src/main/params/units.md [hunk 5]
```diff
@@ -19,62 +19,62 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
 
 | Name        | Abbreviation | Description                                                                          |
 | ----------- | ------------ | ------------------------------------------------------------------------------------ |
-| angstrom    | Angstrom          | angstrom|
-| alpha       | alpha             | alpha    |
-| ampere       | A                | ampere    |
+| angstrom    | Angstrom          | unit of length  |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: unit of length (10^-10 m)

File: docs/src/main/params/units.md [hunk 6]
```diff
@@ -19,62 +19,62 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
-| hertz       | Hz                | frequency |
-| henry       | henry             | henry  |
-| hour        | h                 | hour  |
-| joule       | J                 | Joule: energy N m |
+| gauss       | gauss             | unit of measurement of magnetic induction  |
+| gram        | g                 | unit of mass or weight |
+| hertz       | Hz                | SI unit of frequency |
+| henry       | henry             | unit of electrical inductance  |
+| hour        | h                 | unit of time  |
+| joule       | J                 | derived unit of energy |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: SI unit of energy

File: docs/src/main/params/units.md [hunk 7]
```diff
@@ -19,62 +19,62 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
 | kelvin      | K                 | Kelvin: temperature with a null point at absolute zero |
-| kilogram    | kg                | kilogram, base unit of mass in SI |
-| kilometer   | km                | kilometers |
+| kilogram    | kg                | base unit of mass in SI |
+| kilometer   | km                | unit of length equal to 1,000 metres |
 | liter       | l                 | liter, metric unit of volume |
-| lm          | lm                | lumen  |
+| lm          | lm                | unit of luminous flux, or amount of light  |
 | lsun        | lsun              | solar luminosity  |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: second column should be Lsun

File: docs/src/main/params/units.md [hunk 8]
```diff
@@ -19,62 +19,62 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
-| mol         | mol               | mole |
+| MJD         | MJD               | Modified Julian Date |
+| mol         | mol               | mole: Unit of amount of substance |
 | month       | month             | month |
 | mmyy        | mmyy              | Month/Year |
 | mu0         | mu0               | magnetic constant |
-| muB         | muB               | bohr magneton |
+| muB         | muB               | moment of electrons is bohr magneton |
 | nanometer   | nm                | nanometers |
 | newton      | N                 | Newton: force |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: SI unit of force

File: docs/src/main/params/units.md [hunk 9]
```diff
@@ -19,62 +19,62 @@ The default unit for `UTCTimeKey` and `TAITimeKey` is `second`. For all the rema
...
-| ohm         | ohm               | ohm |
+| ohm         | ohm               | ohm - unit of electric resistance |
 | pascal      | Pa                | Pascal: pressure |
```
Comment (jasonweiss) [inline docs/src/main/params/units.md]: SI unit of pressure

**Review by jasonweiss — COMMENTED** [review]
Overall, it looks much better.  I made a few suggestions for improvements.

**Comment by rajnov08** [comment]
@jasonweiss @kgillies I have made all the changes in csw(master) and esw(rj/CSW-184) branch as suggested. If this looks ok I can merge this PR on esw-ts too.

**Comment by dencoded** [comment]
The units do not exactly match the ones declared in CSW and are instantiated differently.

Closing this PR with a reference to https://tmt-project.atlassian.net/browse/OSWISSUE-63

**Pull request closed without merging**